test/main/modules.js

```diff
@@ -49,6 +49,33 @@ QUnit.test( "does not run before subsequent tests", function( assert ) {
 	assert.equal( this.beforeCount, 1, "beforeCount did not increase from last test" );
 } );
 
+QUnit.module( "before (skip)", {
+	before: function( assert ) {
+		assert.ok( true, "before hook ran" );
+
+		if ( typeof this.beforeCount === "undefined" ) {
+			this.beforeCount = 0;
+		}
+
+		this.beforeCount++;
+	}
+} );
+
+QUnit.skip( "skipped, but should still run before hook", function( assert ) {
+	assert.expect( 1 );
+	assert.ok( true );
```

Comment:
this should not be necessary. Some throwing expression here, like throw 'foo' here would help better, or even `assert.ok( false );`